src/components/CreateGoalForm/Dropdown.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { SelectCountry } from 'react-native-element-dropdown';
+import Searchbar from './SearchBar';
+import Radio from './RadioGroup';
+
+const local_data = [
+  {
+    value: 'short',
+    lable: 'Short Term',
+  },
+  {
+    value: 'long',
+    lable: 'Long Term',
+  },
+];
+
+export interface Props {
+}
+
+const SelectCountryScreen: React.FC<Props> = () => {
+  const [duration, setDuration] = useState('1');
+
+  return (
+    <View>
+      <SelectCountry
+        style={styles.dropdown}
+        selectedTextStyle={styles.selectedTextStyle}
+        placeholderStyle={styles.placeholderStyle}
+        maxHeight={200}
+        value={duration}
+        data={local_data}
+        valueField="value"
+        labelField="lable"
+        placeholder="Select Duration"
+        onChange={(e) => {
+          setDuration(e.value);
+        }}
+      />
+
+      {duration === 'long' && (
+        <View>
+          <Text style={styles.formchild}>Roadmap Type</Text>
+          <Radio />
+
+          <Text style={styles.formchild}>Existing Roadmap</Text>
+          <Searchbar />
+        </View>
+      )}
+
+      {duration === 'short' && (
+        <View>
+          <Text style={styles.formchild}>Associated Long Term Goal</Text>
+          <Searchbar />
+        </View>
+      )}
```

Comment:
Just a suggestion, you can use a ternary operator here as there are only two durations. It would make it a bit cleaner

Reply:
Sure.